kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/ColumnMappingSuite.scala

```diff
@@ -203,4 +500,26 @@ class ColumnMappingSuite extends AnyFunSuite {
     VectorUtils.stringArrayValue(Collections.emptyList()),
     Optional.empty(),
     VectorUtils.stringStringMapValue(Collections.emptyMap()))
+
+  private def assertColumnMapping(
+                                   field: StructField,
+                                   expId: Long,
+                                   expPhysicalName: String = "UUID"): Unit = {
```

Comment:
me again :( indentation. Agreed we need to implement the java formatter. and, in this caese, a scala formatter for this project's tests, too.

Reply:
in the meantime this is probably an incorrectly configured setting in your IDE, the continuation indent

Reply:
sorry I haven't found any docs on how the Scala formatter should be configured for this project. What's the right indentation that should be used here?
Also +1 on configuring a Scala formatter (which I can also take a look at)